.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/github-workflow.json
+
+name: CI
+
+on: [push]
```

Comment:
we should limit this on pushes on main or PRs so that there is no risk on hitting the limits (Not that I think we ever would, but feels like good practice)
https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits